cli/pack/deb_control_dir.go

```diff
@@ -55,11 +68,36 @@ func initControlDir(destDirPath string, ctx *context.Ctx) error {
 			return project.InternalError("Failed to get next Tarantool major version: %s", err)
 		}
 
-		debControlCtx["Depends"] = fmt.Sprintf(
-			"tarantool (>= %s), tarantool (<< %s)",
-			minTarantoolVersion,
-			maxTarantoolVersion,
-		)
+		addDependency(&debControlCtx, common.PackDependency{
+			Name: "tarantool",
+			Relations: []common.DepRelation{
+				{
+					Relation: ">=",
+					Version:  minTarantoolVersion,
+				},
+				{
+					Relation: "<<",
+					Version:  maxTarantoolVersion,
+				},
+			},
+		})
+	}
+
+	// Parse and add dependencies
+	if len(ctx.Pack.Deps) != 0 {
```

Comment:
This action should be performed on context preparing. 
The same for `rpm`.

You can do it on `pack.Run` and save somethere to `ctx.Pack.ParsedDeps`.
Or, you can perform this in `commands.pack`. Just parse specified flags to local variables (`deps` and `depsFile`), then call `common.ParseDependencies` and save the result to `ctx.Pack.Deps`.